src/openrct2/park/ParkFile.cpp

```diff
@@ -375,6 +375,14 @@ namespace OpenRCT2
                             }
                         }
                     });
+
+                // Append guest names object to old park files
+                if (version <= 36)
```

Comment:
Can you define a constant in `ParkFile.h` and use it here? Makes it easier to keep track of changes like these.